File: sorting/bubbleSort/BubbleSort.java
import java.util.Arrays;

public class BubbleSort {
  public static void main(String[] args) {
    int[] arr = {1, 3, 2};
    bubbleSort(arr);
    System.out.println("Sorted" + " " + Arrays.toString(arr));
  }

  public static void bubbleSort(int[] arr) {
      for (int i = 0; i < arr.length - 1; i++) {
         boolean swapped = false;
        for (int j = 0; j < arr.length-1 - i; j++) {
          if (arr[j] > arr[j+1]) {
              int temp = arr[j];
              arr[j] = arr[j+1];
              arr[j+1] = temp;
              swapped = true;
          }
        }
        if (!swapped) break;

      }
  }
}

File: sorting/bubbleSort/bubbleSort.js
class bubbleSort{
    constructor(arr){
        this.arr = arr
    }

    sort(){
        for(let i=0;i<this.arr.length-1;i++){
            let swapped=false
            for(let j=0;j<this.arr.length-1-i;j++){
                if(this.arr[j]>this.arr[j+1]){
                    let temp=this.arr[j]
                    this.arr[j]=this.arr[j+1]
                    this.arr[j+1] = temp
                    swapped=true
                }
            }
            if(!swapped) break;
        }
        return this.arr
    }
}

let arr = [4,3,5,1,0,2,6,1]

const bsort = new bubbleSort(arr)

sorted_arr = bsort.sort()

console.log(sorted_arr)